js/widgets/widgetWindows.js

```diff
@@ -392,7 +392,10 @@ class WidgetWindow {
             siblings[i].style.zIndex = "0";
             siblings[i].style.opacity = ".7";
         }
-        this._frame.style.zIndex = "1";
+
+        // this._frame.style.zIndex = "1";
```

Comment:
No need to keep this old value in a comment. We can always look at the commit history.